### PR TITLE
Fix: Fix underline issue for login and sign up buttons

### LINKED ISSUE #177: Bug: text underline on the Login & Sign Up (blue) buttons
**Describe the bug**
There is a text underline on the Login & Sign Up (blue) buttons. 

**To Reproduce**
1. Go to http://localhost:3000/login
2. See error

**Expected behavior**
Remove text underline from the login & sign up (blue) buttons

**Screenshots**
image

image

**Desktop (please complete the following information):**
 - OS: Any
 - Browser: Any
 - Version: Any

**Additional context**
I think setting `text-decoration: none` would work.

### PR DESCRIPTION
**Pull request by sachinsom93**
### Description

Removed the unnecessary underlines under login and sign Up buttons

Fixes #177 

### Type of Change:

- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
Register page
Login page

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/login/Login.jsx [hunk 1]
```diff
@@ -92,7 +92,13 @@ export default function Login() {
                         </div>
                         <div className="row button-group">
                             <div className="col">
-                                    <a className="btn btn-primary" id="registerbtn" href="/register" role="button">Sign Up</a>
+                                <Link 
+                                    to="/register"
```
Comment (nlok5923) [inline src/login/Login.jsx]: please remove unnecessary changes.

File: src/register/Register.jsx [hunk 2]
```diff
@@ -254,7 +254,13 @@ export default function Register() {
                         </div>
                         <div className="row button-group">
                             <div className="col">
-                                    <a className="btn btn-primary" id="loginbtn" href="/login" role="button">Login</a>
+                                <Link 
+                                    to="/login"
```
Comment (nlok5923) [inline src/register/Register.jsx]: here also

**Review by nlok5923 — CHANGES_REQUESTED** [review]
please refer to the comments and also i think you should not compare package-lock.json  what's your view @mtreacy002 ma'am.

File: src/login/Login.jsx (hunk 1 above)
Comment (sachinsom93) [reply src/login/Login.jsx]: Thanks for review this @nlok5923 , but I think it is better to use Link instead of anchor tags to navigate to some other route, what's your view @mtreacy002 ma'am, should I revert this change ?

Comment (Rahulm2310) [reply src/login/Login.jsx]: @sachinsom93 @nlok5923  It is better to use `Link` in `react` as anchor tags trigger a page refresh which would reset the application states whereas the `Link` and `Navlink` of react-router doesn't trigger a page refresh. In single page apps with `react`, it is better to use `Link` from `react-router`.

**Comment by Rahulm2310** [comment]
@sachinsom93 Please remove package-lock.json from the commit as it has unnecessary changes which need not to be pushed.

File: src/login/Login.jsx (hunk 1 above)
Comment (haelmj) [reply src/login/Login.jsx]: @nlok5923 Just as an extra opinion, I agree with @sachinsom93 and @Rahulm2310 on the use of `Link` tags in place of `a` in React for the same reasons specified above. Generally speaking, the `Link` tags are better placed for in-app navigation(relative urls) and the `a` tags for external links(absolute urls).

**Comment by mtreacy002** [comment]
@sachinsom93 , can you please give us an update on this?

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@sachinsom93 , as per @Rahulm2310 suggested, can you please remove this package-lock.json from the commit (not to delete it from your local)?

**Comment by sachinsom93** [comment]
> @sachinsom93 , as per @Rahulm2310 suggested, can you please remove this package-lock.json from the commit (not to delete it from your local)?

@mtreacy002 I will do it as soon as possible.

**Comment by HeroicHitesh** [comment]
> @mtreacy002 I will do it as soon as possible.

@sachinsom93 Can you please share updates on your progress?

File: package.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^3.4.3"
+    "react-scripts": "^4.0.3"
```
Comment (mtreacy002) [inline package.json]: is this update needed, @sachinsom93 ? Do you have any reason why we need this? This is also the reason why the package-lock.json is changed.

Comment (sachinsom93) [reply package.json]: @mtreacy002 , I didn't update it manually, it got updated when I use "npm install" at the time of project setup.

Comment (Rahulm2310) [reply package.json]: @sachinsom93 We already have an issue open for updating the react-scripts version (#194). So, please revert these changes to the package.json and package-lock.json here.

**Comment by Rahulm2310** [comment]
>@sachinsom93 , as per @Rahulm2310 suggested, can you please remove this package-lock.json from the commit (not to delete it from your local)?

@sachinsom93 you were requested to not include package-lock.json in the commit (not to delete package-lock.json). Also now I am unable to see your changes to the buttons. 

The earlier changes you made to the buttons were correct. We just wanted you to not push unnecessary changes ( made to package.json and package-lock.json). If you have any doubts, then please ask.

**Comment by mtreacy002** [comment]
@sachinsom93 , can you please give us an update on this?

**Comment by sachinsom93** [comment]
> @sachinsom93 , can you please give us an update on this?

@mtreacy002, sorry for being inactive, I tried to revert the changes but it didn't work. So, I am going to create new pr for the same. 
Will it work ?

**Comment by vj-codes** [comment]
@sachinsom93 you can make a new PR 
Also, your fork is 7 commits behind, just make sure it's in sync with develop branch before creating a new one:)
